bee-node/build.rs

```diff
@@ -36,3 +89,53 @@ fn main() -> Result<(), BuildError> {
 
     Ok(())
 }
+
+#[cfg(feature = "dashboard")]
+mod dashboard {
+    use super::*;
+
+    use checksums::Algorithm;
+    use zip::ZipArchive;
+
+    use std::path::Path;
+
+    const RELEASE_URL: &str =
+        "https://github.com/iotaledger/node-dashboard/releases/download/v1.0.0/node-dashboard-bee-1.0.0.zip";
+    const RELEASE_CHECKSUM: &str = "d61bce7d70b51ea88536b22c5341ccdc3c4651bf8e02e093a771312f3f046c30";
+
+    pub(super) fn fetch<P: AsRef<Path>>(dashboard_dir: P) -> Result<(), BuildError> {
+        println!("downloading latest dashboard release from {}", RELEASE_URL);
+
+        let client = reqwest::blocking::Client::builder()
+            .user_agent("bee-fetch-dashboard")
+            .build()
+            .expect("could not create client");
+
+        let mut tmp_file = tempfile::NamedTempFile::new().expect("could not create temp file");
+
+        client
+            .get(RELEASE_URL)
+            .send()
+            .and_then(|resp| resp.error_for_status())
+            .map_err(|e| BuildError::DashboardRequest(e.status().map(|code| code.as_u16()), RELEASE_URL.to_string()))?
+            .copy_to(&mut tmp_file)
```

Comment:
Just to confirm: this is streaming from the client to the file, right ? Not first downloading then writing ?

Reply:
that's what I was curious about, too, but I can also confirm from the reqwest docs.